.gitignore

```diff
@@ -37,6 +36,13 @@ desktop.ini
 # server logs
 /logs/osu/*
 
+# Logs
+logs
+*.log
```

Comment:
Not sure if we want to do this. (also the line above)

Reply:
yeah, these should probably be for specific paths rather than this broad.